src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [visibleMovies, setVisibleMovies]
+  = useState(moviesFromServer.map(el => el));
```

Comment:
```suggestion
  = useState(moviesFromServer);
```
you can just simply do this